spec/controllers/auth/sessions_controller_spec.rb

```diff
@@ -142,7 +140,7 @@
         end
 
         it 'sends a suspicious sign-in mail' do
-          expect(UserMailer).to have_received(:suspicious_sign_in).with(user, current_ip, anything, anything)
+          expect(UserMailer.deliveries.size).to eq(1)
```

Comment:
This test too could benefit from testing the recipient address and the subject line!

Reply:
pushed again with even more subject checking